Homework 2/TwitterService/src/main/java/com/example/twitterservice/TwitterController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.twitterservice;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+import twitter4j.*;
+
+    // Umang`s Spring Boot Controller
+
+@Controller
+public class TwitterController {
+
+    @GetMapping("/")
+    public ModelAndView home() {
+        return new ModelAndView("index");
+    }
+
+    @PostMapping("/tweet")
+    public ModelAndView tweet(@RequestParam String tweetText) throws TwitterException {
+
+        //  Gerardo`s logic
+
+        Twitter twitter = new TwitterFactory().getInstance();
+        final TwitterV2 v2 = TwitterV2ExKt.getV2(twitter);
+        final CreateTweetResponse createTweet = v2.createTweet(
+                null,
+                null, null, null, null, null, null,
+                null, null, null, null, tweetText);
+        ModelAndView modelAndView = new ModelAndView("index");
+        modelAndView.addObject("tweetId", createTweet.getId());
+        return modelAndView;
+    }
+
+    @PostMapping("/delete")
+    public ModelAndView delete(@RequestParam long tweetIdToDelete) throws TwitterException {
```

Comment:
Suggestion: We should either have a @ControllerAdvice handling TwitterException.